book/starting-utils.md

```diff
@@ -49,21 +49,33 @@ Usage:
 Specific information about what these commands do is given in the following sections and on the [*Solo CLI* README](https://github.com/3drobotics/solo-cli).
 
 
-## Connecting to Solo and the Internet
+## Connecting Solo to the Internet
 
-During development, you will most often want access to Solo and the Internet simultaneously. The command `solo wifi` connects your Controller to a local wifi network, allowing you to develop for Solo while also enabling connection to the outside Internet. The steps for doing so are as follows:
+The `solo wifi` command connects your Controller to a local WiFi network. Solo uses this connection (via the Controller network) to access the Internet during development and to [download and install packages](starting-installing.html#installing-packages).
 
-* Connect your PC to the Controller's WiFi network.
-* Run `solo wifi --name=<ssid> --password=<password>` command from your PC's command line. The SSID and password should be those of a local WiFi network, i.e. that of your home or your office.
-* If your PC does not yet have an Internet connection, disconnect and reconnect to Solo's WiFi network at this time.
+<aside class="tip">The development PC still connects to Solo and the Controller by their dedicated IP addresses on the Controller Network (`10.1.1.1` and `10.1.1.10`) rather than the IP of the Controller on your local WiFi network.
```

Comment:
I would remove the part about the IP of the Controller in the local Wifi network, developer never sees that and doesn't need to know it, for me it only adds more confusion. I would rewrite this as:
```
The development PC still needs to connect to the Controller's WiFi network and not your WiFi network. You also continue to use the Solo and Controller dedicated IP addresses (`10.1.1.1` and `10.1.1.10`).